apps/www/lib/redirects.js

```diff
@@ -2318,4 +2318,9 @@ module.exports = [
     source: '/blog/supabase-ai-content-storm',
     destination: 'https://dev.to/supabase/ai-ignites-the-rain-content-storm-is-back-kdl',
   },
+  {
+    permanent: true,
+    source: '/docs/guides/functions/global-deployments',
+    destination: 'docs/guides/functions',
+  },
```

Comment:
Just noticed you also have this PR https://github.com/supabase/supabase/pull/18617/files

Perhaps we let that one merge first, and then set this redirect to `/guides/functions/regional-invocation`? Seems somewhat closely related.

```suggestion
  {
    permanent: true,
    source: '/docs/guides/functions/global-deployments',
    destination: 'docs/guides/functions/regional-invocation',
  },
```

Reply:
yeah, sounds good.